src/App.vue

```diff
@@ -39,5 +65,51 @@ export default defineComponent({
       padding: $page-padding-regular;
     }
   }
+  &__error {
+    position: sticky;
+    top: 0;
+    width: 100%;
+    z-index: 100;
+    background-color: $red-cherryred;
+    animation: error-message-animation ease;
+    animation-duration: v-bind(errorTime);
+  }
+  &__error-info {
+    display: block;
+    margin: 0;
+    padding: 1.6rem;
+    text-align: center;
+    color: $white;
+  }
+  &__error-escape-button {
+    position: absolute;
+    top: 50%;
+    right: 2rem;
+    transform: translateY(-50%);
+    background-color: $red-cherryred;
+    border: none;
+    color: $white;
+    cursor: pointer;
+  }
+
+  @keyframes error-message-animation {
+    0% {
+      opacity: 0;
+    }
+    15% {
+      opacity: 1;
+    }
+    70% {
+      opacity: 1;
+    }
+    100% {
+      opacity: 0;
+    }
+  }
 }
 </style>
+
+<!-- TypeScript part 2 notatki
+
+
+-->
```

Comment:
I think this shouldn't be here right 😄 ?

Reply:
Oh of course, I don't know how did i miss that... I'm sincerely sorry 😕